src/shared/telemetry/activation.ts

```diff
@@ -83,13 +105,18 @@ export function isTelemetryEnabled(toolkitSettings: SettingsConfiguration): bool
         return false
     }
 
-    // Current value is expected to be a boolean
-    if (typeof value === 'boolean') {
-        return value
+    //Will throw an error if type is not a boolean
+    try {
+        if (typeof value !== 'boolean') {
+            throw new TypeError(`${value} is not a boolean`)
```

Comment:
The option where we throw was suggesting that the exception would escape uncaught.

Honestly the safest thing to do would be to only change `Enabled` and `Disabled` to `true` and `false` resp. Would leave all other unexpected values as-is as it's the least surprising to the user.

If an unexpected value is read, I would treat it as an attempt to opt out and return `false` here. This setting is special in that assuming a `true` value could lose user trust whereas assuming a `false` value cannot. Hence why I said it was the safer of the two options.

Reply:
I agree that we should only change the user setting if it exactly matches 'Enable' or 'Disable'.  I will go ahead and make that change.  As for the default for unexpected values, I know there has been a change in this recently.  Let me get back to you with more input.